README.md

```diff
@@ -19,6 +19,8 @@ either plain Bazel or [Bazelisk](https://github.com/bazelbuild/bazelisk).
 
 ### Building and running the test suite
 
+`./examples/run_djinni.sh` generates the examples sources.
```

Comment:
There are also `run_djinni.sh` scripts in other directories.  Maybe let's just say "use the run_djinni.sh scripts to generate bridge code"

Reply:
Done